contrib/face_recognition.lua

```diff
@@ -153,6 +154,7 @@ local function reset_preferences()
   fc.unknown_tag.text = "unknown_person"
   fc.no_persons_found_tag.text = "no_persons_found"
   fc.ignore_tags.text = ""
+  fc.category_tags.text = "People"
```

Comment:
I think this should be an empty string so that it acts just like the current version when reset.  This takes into account the people who have been using this and have set up their reference face files with a tag hierarchy.